addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -473,7 +487,7 @@ const UserValueWidget = Widget.extend({
         // If the widget has no associated method, it should not notify user
         // value changes
         if (!this._methodsNames.length) {
-            return;
+            console.warn('UserValueWidget with no methods notifying value change');
```

Comment:
@qsm-odoo you didn't ask about this but I may as well tell you about it since it's not obvious. I did this to avoid having to pass fake method names to manually created UVWs, and then having to call `loadMethodsData` on them. I don't think it should break any existing code, since the rest of the existing code interacts with UVWs through methods but since there are no methods the rest of this function will be a no-op (unless a parent widget catches the notification event, which is what I've done here)

Reply:
I did not ask because I guessed the reason ;)